datafusion/functions-array/Cargo.toml

```diff
@@ -38,7 +38,8 @@ path = "src/lib.rs"
 
 [dependencies]
 arrow = { workspace = true }
-datafusion-common = { workspace = true, default-features = true }
+arrow-array = { workspace = true }
```

Comment:
I wonder why we need to add arrow-array explicilty? Isn't everything we need already exported via `arrow`?

Reply:
I will remove it. I was trying to use some methods in that crate so imported that, I will remove it